Jellyfin.Api/Controllers/SyncPlayController.cs

```diff
@@ -73,24 +73,58 @@ public class SyncPlayController : BaseJellyfinApiController
         public ActionResult SyncPlayJoinGroup(
             [FromBody, Required] JoinGroupRequestDto requestData)
         {
-            var currentSession = RequestHelpers.GetSession(_sessionManager, _authorizationContext, Request);
+            var joiningSession = RequestHelpers.GetSession(_sessionManager, _authorizationContext, Request);
+
+            // TODO: check if session can control remote session.
+            if (requestData.RemoteSessionId != null)
+            {
+                var remoteSession = _sessionManager.GetSession(requestData.RemoteSessionId);
+                if (!remoteSession.Capabilities.SupportsSyncPlay)
+                {
+                    // Remote session does not support SyncPlay.
+                    return NoContent();
```

Comment:
Should this be `BadRequest` so the client knows it didn't do anything?

Reply:
That's right. Which reminds me that this needs to be handled client side as well.